src/NovaInlineRelationship.php

```diff
@@ -129,6 +129,23 @@ protected function resolveResourceFields($resource, $attribute, $properties)
         $this->updateFieldValue($resource, $attribute, $properties);
     }
 
+    /**
+     * Pluck id's from the model or collection.
+     *
+     * @return array
+     */
+    protected function modelIds()
+    {
+        if ($this->value instanceof Model) {
+            $models = [$this->value->{$this->value->getKeyName()}];
+        } elseif ($this->value instanceof Collection) {
+            $key = $this->value->first()->getKeyName();
+            $models = $this->value->pluck($key)->all();
```

Comment:
I think we need to check for empty collection as well

Reply:
@navneetrai Yep, just pushed a fix for that